### ISSUE TITLE
Error: object 'design' not found

### ISSUE THREAD

**Issue by njcorline** [jim-breck/FB4#28]
Hello, 

I downloaded the most recent version of fishbioenergetics 4.0 (FB4 v1.1.5) and I am try to work through the first suggested exercise in the documentation, but the output results in the error message "Error: object 'design' not found". The input files are all reading fine. This could be due to something on my end but I think this may be due to the recent update. 

Cheers!

**Comment by njcorline** [comment]
Sorry, I meant to type FB4 v1.1.6

**Comment by ojensen1** [comment]
Yes, I'm having the same issue as njcorline.  When I use FB4 v1.1.5 I don't get this "Error: object 'design' not found", so it seems to be something in the new version 1.1.6.  UseDesignFile <- FALSE on line 70 in v1.1.6 but for some reason it's still looking for a Design File

**Comment by UVM-Jason** [comment]
I am also having this problem.
